sdk/servicebus/Azure.Messaging.ServiceBus/api/Azure.Messaging.ServiceBus.netstandard2.0.cs

```diff
@@ -426,6 +426,36 @@ public enum ServiceBusTransportType
         AmqpWebSockets = 1,
     }
 }
+namespace Azure.Messaging.ServiceBus.Amqp
+{
+    public enum AmqpBodyType
+    {
+        Unspecified = 0,
+        Data = 1,
+        Sequence = 2,
+        Value = 3,
+    }
+    public static partial class ServiceBusMessageExtensions
+    {
+        public static Azure.Messaging.ServiceBus.Amqp.AmqpBodyType GetAmqpBodyType(this Azure.Messaging.ServiceBus.ServiceBusMessage message) { throw null; }
```

Comment:
Why are these extensions and not instance members on the message?

Reply:
This is an advanced scenario that 99% of users would not care about. That is why we are using extension methods.